src/compiler/utilities.ts

```diff
@@ -1217,16 +1217,6 @@ namespace ts {
                         traverse(operand);
                     }
                     return;
-                case SyntaxKind.EnumDeclaration:
-                case SyntaxKind.InterfaceDeclaration:

```

Comment:
It's still worth skipping type-only declarations like `InterfaceDeclaration` and `TypeAliasDeclaration` since walking those ASTs would be unnecessary work.

Reply:
Done, with a slightly revised comment.